src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -492,31 +496,15 @@ CAC.Map.Control = (function ($, Handlebars, cartodb, L, _) {
             // get the points from the edited linestring
             var modified = editLayer.toGeoJSON().features[0].geometry.coordinates;
             cleanUpItineraryEditEnd(false);
-            endItineraryEdit(itinerary.getTurnPoints(), modified);
+            // remove start and end points
+            modified.shift();
+            modified.pop();
```

Comment:
I seem to have an aversion to `shift` and `pop` in null context.  Not sure if it's rational or not.  Lodash `slice` supports negative indexes, though, so this could be simplified to
```
// get the points from the edited linestring, excluding start and end
var modified = _.slice(editLayer.toGeoJSON().features[0].geometry.coordinates, 1, -1);`
```